### PR TITLE
Fixing BZ1291521 where long project name spills out of modal

### PR DESCRIPTION
**Pull request by rhamilto**
@spadgett or @jwforres, review and merge, please?

Before:

screen shot 2016-03-09 at 10 45 22 am

After:

screen shot 2016-03-09 at 10 45 03 am

### CODE REVIEW

File: assets/app/styles/_core.less [hunk 1]
```diff
@@ -775,6 +775,7 @@ a.disabled-link {
     font-size: 21px;
     font-weight: 500;
     margin-bottom: 20px;
+    word-wrap: break-word;
```
Comment (spadgett) [inline assets/app/styles/_core.less]: Probably should use our work-break mixin.

```
word-break();
```

Comment (spadgett) [inline assets/app/styles/_core.less]: Although maybe only if we merge #7883 for 1.2.

Comment (spadgett) [inline assets/app/styles/_core.less]: #7883 is in the merge queue. @rhamilto can you switch to use the mixin?

**Comment by rhamilto** [comment]
@spadgett, switched to the mixin.  The mixin seems like overkill to me.  Why use four declarations when one will do?  cc:  @sg00dwin

**Comment by spadgett** [comment]
@rhamilto If it's overkill we should fix the mixin or get rid it. There was an issue at one time in Firefox it was there to address. I see there is a -ms prefixed one, not sure if we still need that.

FWIW, it's down to 3 styles now :)

**Comment by spadgett** [comment]
Merging this fix. If we change the mixin, I think it should be in a follow-on PR and fix the places it's used.

[merge]

**Comment by sg00dwin** [comment]
I think it can be reduced to 2
word-wrap: break-word; // firefox, legacy
overflow-wrap: break-word; // updated spec

**Comment by spadgett** [comment]
If it's only those two, we could remove the mixin when we add PostCSS.

**Comment by spadgett** [comment]
@rhamilto bindata conflict

**Comment by rhamilto** [comment]
@spadgett, I resolved the bindata conflict.

**Comment by spadgett** [comment]
thanks, [merge]

**Pull request merged**